### ISSUE TITLE
Stuck at "running in a929859...."

### ISSUE THREAD

**Issue by burkeholland** [oren/docker-nativescript#2]
Excuse my ignorance on Docker, but after running `docker build -t nativescript .` I get

```
 ---> Running in a7218635d5f3
```

And the terminal just sits there. Is that expected? Seems like I should get back to the command prompt after the thing has booted up.

**Comment by oren** [comment]
what OS are you using? if it's a Mac, i would recommend using docker machine. I am on ubuntu so I can't really troubleshoot Mac related issues ):

**Comment by burkeholland** [comment]
Yeah - on a Mac. I have no idea what Docker Machine is, but I'll look into it and report back. Sorry for the lag on this.
